junit5/README.md

```diff
@@ -0,0 +1,243 @@
+# Weld JUnit 5 Extension
+
+This extension uses the extension mechanism introduced in JUnit 5.
+Therefore, in order to use this extension in your test, you have to annotate your test class with `@ExtendWith(WeldJunit5Extension.class)`.
+It will automatically inject into all your `@Inject` fields in given test instance and start/stop Weld SE container based on your configuration.
+Even if no configuration is provided, the default one will kick in.
+Supports both test lifecycles - per method and per class.
+
+Requires JUnit 5 and Java 8.
+
+## Table of contents
+
+* [JUnit 5](#junit-5)
+  * [WeldInitiator](#weldinitiator)
+    * [Flat Deployment](#flat-deployment)
+    * [Convenient Starting Points](#convenient-starting-points)
+      * [Test class injection](#test-class-injection)
+      * [Activating context for a normal scope](#activating-context-for-a-normal-scope)
+      * [Adding mock beans](#adding-mock-beans)
+
+## JUnit 5
+
+```xml
+<dependency>
+  <groupId>org.jboss.weld</groupId>
+  <artifactId>weld-junit5</artifactId>
+  <version>${version.weld-junit}</version>
+</dependency>
+```
+
+### WeldInitiator
+
+`org.jboss.weld.junit5.WeldInitiator` is an entry point you will want to define if you wish to customize how we bootstrap Weld.
```

Comment:
We should mention that for junit5 `WeldInitiator` is not needed at all. Probably add a `Basic Example` section before this one and explain the default behavior?

Reply:
Good point, will come up with something